.github/workflows/create-pullrequest-prerelease.yml

```
@@ -52,8 +52,13 @@ jobs:
         uses: marocchino/sticky-pull-request-comment@v2
         with:
           message: |
-            wrangler prerelease is available for testing:
+            A wrangler prerelease is available for testing. You can install it in your project with:
 
             ```sh
             npm install --save-dev https://prerelease-registry.developers.workers.dev/runs/${{ github.run_id }}/wrangler
             ```
+
+            Or you can try it out directly with:
```

I was a bit surprised that this was `dev` specific, since there are lots of other uses for `wrangler`...

```suggestion
            Or you can try developing a worker directly with:
```

---

No reason, I just had to choose one of many commands haha.